### DISCUSSION TITLE
Sierra iso

### DISCUSSION THREAD

**Discussion by erikdubois** [picodotdev/alis#228] [Ideas]
I was wondering if you have plan to incorporate zfs.
Seems a hot topic these days.

Started a new iso called Sierra.

Installs Xfce and Chadwm and all the apps I want

**Comment by arcolinuxz** [comment]
is it possible to skip this question?

This is exactly what I want.

![yesno](https://user-images.githubusercontent.com/36679406/209312400-3361018c-8cf8-4c8a-9a77-4afe3419957a.png)

**Comment by picodotdev** [comment]
Hello Erik,

ZFS seems too experimental and requires many steps to make it work, I am not wishing to add a feature that potentially breaks the system, there is already support for BTRFS that offers additional features over ext4.

You should be able to omit the continue confirmation using this option (let me know if not):

`./alis.sh -w`

**Comment by erikdubois** [comment]
ok thanks 

understood

**Reply by erikdubois** [reply]
alis.sh -w works like a charm

**Comment by erikdubois** [comment]
Comparing our githubs... now to get to a code that works

![image](https://user-images.githubusercontent.com/10594806/210163663-7a440180-ee7d-4c69-82f9-b0ee1c7f7933.png)


This is where It gets stuck

**Comment by erikdubois** [comment]
back to the basics - I believe this is in the past ;-)
